### PR TITLE
Remove GodMode in Floor type-11 and automate building process.

### PR DESCRIPTION
**Pull request by EstebanZapataME**
Vanilla removes IDDQD cheat when finishing E1M8 for example, but this source port doesn't have that.

Also made it so that you can more easily build the rom by just opening msys2.bat, instead of also having to type "make"

### CODE REVIEW

**Comment by doomhack** [comment]
Looks good to me. Thank you.

File: source/p_spec.c [hunk 1]
```diff
@@ -2172,6 +2172,7 @@ void P_PlayerInSpecialSector (player_t* player)
 
       case 11:
         if (!(_g->leveltime&0x1f))
+          _g->player.cheats -= CF_GODMODE; //Vanilla disables GodMode in floor-11
```
Comment (FrenkelS) [inline source/p_spec.c:2175]: I think this if-statement is missing curly braces.

It looks a bit different in the original Linux Doom source:
https://github.com/id-Software/DOOM/blob/77735c3ff0772609e9c8d29e3ce2ab42ff54d20b/linuxdoom-1.10/p_spec.c#L1054

**Comment by Kippykip** [comment]
Huh yeah I guess the original has player->cheats &= ~CF_GODMODE; instead
Not sure if it matters though

**Comment by FrenkelS** [comment]
I mean in the original code `cheats &= ~CF_GODMODE;` isn't behind an if-statement, while `P_DamageMobj (player->mo, NULL, NULL, 20);` is.
But here it's the other way around.

**Comment by FrenkelS** [comment]
Come to think of it, there's a difference between `cheats &= ~CF_GODMODE` and  `cheats -= CF_GODMODE`. The first one turns of the GodMode cheat flag,
If you don't have any cheats activated, `cheats` is `0,` then `cheats -= CF_GODMODE` will make `cheats == -2` which is equal to `11111110b` with a lot more leading 1s, which turns on the GodMode cheat flag, and NoMomentum and EnemyRockets.

**Pull request merged**